index.html

```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css "href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title></title>
+  </head>
+
+  <body>
+    <div class="box">
+      <h2>Sign in</h2>
+      <form>
+        <div class = "form-group">
+          <input type="user" class="form-control" id="exampleInputUser1" placeholder="User Name" >
```

Comment:
Fix id naming. The id of this input should be clearly `username`. Also username is one word. Replace `User Name` with `username`